tests/python/openassetio/managerAPI/test_managerinterface.py

```diff
@@ -50,6 +50,20 @@ def test_when_overridden_then_returns_value(self, manager_interface):
         assert manager_interface.displayName() == "Stub Manager"
 
 
+class Test_ManagerInterface_info:
+    def test_when_not_overridden_then_returns_empty_dict(self):
+        info = ManagerInterface().info()
+
+        assert isinstance(info, dict)
+        assert info == {}
+
+    def test_when_overridden_then_returns_expected_dict(self, manager_interface):
+        info = manager_interface.info()
+
+        assert isinstance(info, dict)
+        assert info == {"stub": "info"}
+
```

Comment:
[fix] Are we relying on automagic binding of `InfoDictionary` to Python via pybind, where it is always converted to a native python `dict`? If so, we need checks for the expected behaviour around:
- Non string keys
- Invalid value types (including None)

Reply:
Don't worry about this for now - as we realised that when we have a C++ `Manager` implementation, we'll be able to return bogus values in the `ValidatingManagerInterface` implementation in this file. To try and do it now needs a lot of scaffolding as currently, a python override totally bypasses C++.